### DISCUSSION TITLE
Database Notifications Not Showing

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#7758] [Help] [answered]
I dont know which blade file i have to change to aply this code regarding the notifications:

To add database notifications to your app, you must add a new Livewire component to your Blade layout:

@livewire('database-notifications')

Any ideas? im saving the notifications in the database but im not being able to show the icon in the top menu.

Thanks in advanced.

**Comment by danharrin** [comment, marked as answer]
If youre using the panel builder, you dont need to. Just add `->databaseNotifications()` to your panel provider

**Reply by jcgmalave** [reply]
Thanks a lot :)
